applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -341,6 +341,19 @@ void KratosGeoMechanicsApplication::Register() {
     KRATOS_REGISTER_VARIABLE( DT_TEMPERATURE )
     KRATOS_REGISTER_VARIABLE( NORMAL_HEAT_FLUX )
 
+    KRATOS_REGISTER_VARIABLE( AIR_TEMPERATURE )
+    KRATOS_REGISTER_VARIABLE( SOLAR_RADIATION )
+    KRATOS_REGISTER_VARIABLE( AIR_HUMIDITY )
+    KRATOS_REGISTER_VARIABLE( PRECIPITATION )
+    KRATOS_REGISTER_VARIABLE( WIND_SPEED )
+    KRATOS_REGISTER_VARIABLE( A1_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( A2_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( A3_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( ALPHA_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( QF_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( SMIN_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( SMAX_COEFFICIENT )
```

Comment:
To me it's not directly clear what these coefficient mean and that they are related to micro-climate. Do we need any more clarification in the name, or is this enough?